### PR TITLE
feat: implement cat callback (MN-310)

### PR DESCRIPTION
**Pull request by deepakglobant**

### CODE REVIEW

File: src/screens/InspectionCapture/index.js [hunk 1]
```diff
@@ -27,16 +30,22 @@ export default function InspectionCapture() {
   const { errorHandler } = useMonitoring();
 
   const { inspectionId, taskName, vehicleType, selectedMode } = route.params;
-  const { sights } = useClient();
+  const { Clients, sights } = useClient();
```
Comment (rohitsharma120582) [inline src/screens/InspectionCapture/index.js:33]: It should be `clients` instead of 'Clients`.

File: src/screens/InspectionCapture/index.js [hunk 2]
```diff
@@ -60,7 +69,28 @@ export default function InspectionCapture() {
         }],
         { cancelable: true },
       );
-    } else { navigation.navigate(names.LANDING, { inspectionId, captureComplete: true }); }
+    } else {
+      const damageDetectionTask = inspection.tasks.find(
+        (task) => task.name === monk.types.TaskName.DAMAGE_DETECTION,
+      );
+
+      if (Clients === ClientParamMap.pd8) {
```
Comment (rohitsharma120582) [inline src/screens/InspectionCapture/index.js:77]: Change this condition to `client === Clients.CAT`.

File: src/screens/InspectionCapture/index.js (hunk 1 above)
Comment (deepakglobant) [reply src/screens/InspectionCapture/index.js]: Done

File: src/screens/InspectionCapture/index.js (hunk 2 above)
Comment (deepakglobant) [reply src/screens/InspectionCapture/index.js]: Done

**Review by rohitsharma120582 — APPROVED** [review]

**Pull request merged**